themes/root/theme.config.php

```diff
@@ -17,7 +17,7 @@
             'VuFind\View\Helper\Root\Captcha' => 'VuFind\View\Helper\Root\CaptchaFactory',
             'VuFind\View\Helper\Root\Cart' => 'VuFind\View\Helper\Root\CartFactory',
             'VuFind\View\Helper\Root\Citation' => 'VuFind\View\Helper\Root\CitationFactory',
-            'VuFind\View\Helper\Root\Component' => 'Laminas\ServiceManager\Factory\InvokableFactory',
+            \VuFind\View\Helper\Root\Component::class => \VuFind\View\Helper\Root\ComponentFactory::class,
```

Comment:
I'm not opposed to changing from strings to class references, but it feels a little weird to do it in this one line and not convert the rest. Maybe it would be best to stick to the existing convention for this PR, and then open a separate one to convert everything (as I think we may have discussed in the past).